app/src/main/res/layout/activity_settings.xml

```diff
@@ -1,19 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/root"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical">
+    android:background="@drawable/launch_screen_background">
```

Comment:
Why are you specifying a background that is different? This changes the appearance of the settings, and creates a weird interface for elements that rely on the previous value. For example, the manage sensors interface now has striping because you've changed the background to an off-white color:
![image](https://user-images.githubusercontent.com/8148535/193402718-0fdd678f-41d9-484f-92e7-7f2c6f08d586.png)

Reply:
After removing it the app appears to be working fine...

Reply:
Tried in on another device with a lower Android version, and now the settings screen is showing the Home Assistant logo in the background due to using the launch screen background...
![image](https://user-images.githubusercontent.com/8148535/193448042-81f6a44b-278f-422e-aebf-1cb38177c8a2.png)

Reply:
> After removing it the app appears to be working fine...

I've just tested the changes where no explicit background is used, and the rootview is retrieved as per your suggestion (`window.decorView.rootView`) and it indeed seems to work just fine. 
Then I'm wondering what the issue was that caused the content to be displayed on top of the blurview, but at least it now behaves as expacted.

Thanks for the feedback! ;)